src/core/components/radio/index.tsx

```diff
@@ -188,11 +203,16 @@ const Radio = ({
 
 const radioGroupDefaultProps = {
 	orientation: "vertical",
+	optional: false,
 	hideLabel: false,
 }
 
 const radioDefaultProps = {
 	disabled: false,
+	type: "radio",
```

Comment:
This is already hardcoded in the `Radio` component's `<input>` element. Where do you think is the best place to define this?